src/magento-payments/onboarding-payments.md

```diff
@@ -1,3 +1,129 @@
 ---
 title: Onboarding Payments
----
\ No newline at end of file
+---
+
+Magento Payments is your turnkey self-service solution---including sandbox testing, fraud prevention, and a super simple setup---for providing robust and secure payment processing for your Magento store.
+
+To get started using Magento Payments you must complete a few onboarding steps to connect your instance with our payments functionality.
+
+1. [Connect your Magento instance](#configure-magento-services) to Magento Software as a Service (SaaS) services. This only needs to be completed once per Magento instance.
+1. [Onboard and set up the sandbox service](#enable-sandbox-testing) (or, alternatively, proceed to [enabling live payments](#enable-live-payments) if you've tested functionality in another environment) to set up a test PayPal payment processing account.
+1. [Enable Magento Payments as your payment method](#set-magento-payments-as-payment-method), in sandbox mode, to start processing test payments in your store.
```

Comment:
The merchant can't process payments until he onboarded, not sure if it needs further clarifications.

Reply:
I don't think it needs further clarification. These steps are listed in the order they should likely be completed. In the next two steps you have to complete merchant onboarding and active payments in live mode to start processing live payments. I think we're okay here.